### DISCUSSION TITLE
The node environment is incorrectly installed

### DISCUSSION THREAD

**Discussion by deno-source** [opensource9ja/danfojs#658] [General]
**Describe the bug**
node environment is too complex to install, and there are various exceptions between versions, which greatly affects the experience and makes people give up using this solution. It is recommended to optimize the installation in two areas: first, to make the browser version support nodejs environment. Second, update the official documentation with details about the installation considerations for each platform in the node environment.

**Comment by risenW** [comment]
@deno-source What issues did face when installing, also please provide a description on how to reproduce so we can help you.
